src/x/__tests__/xNoiseSanPlot.test.ts

```diff
@@ -1021,10 +1021,10 @@ test('get noise level', () => {
     ],
   };
 
-  const noise = xNoiseSanPlot(data.re, { magnitudeMode: false });
+  const noise = xNoiseSanPlot(data.re, { magnitudeMode: false, mask: [] });
   const noiseFromMagnitude = xNoiseSanPlot(reimAbsolute(data), {
     magnitudeMode: true,
   });
   expect(noise.snr).toBeCloseTo(56.8, 1);
-  expect(noiseFromMagnitude.snr).toBeCloseTo(46.4, 1);
+  expect(noiseFromMagnitude.snr).toBeCloseTo(40.2, 1);
```

Comment:
Is it normal that the result changes?

Reply:
yes, because before there was and error in the calculation of the rayleigh inverse CDF, I did test the function responsible of this value, and the results match with Matlab an wikipedia now.